eap-session-replication/README.md

```diff
@@ -0,0 +1,121 @@
+Create a clustered JBoss EAP application with session replication
+===================
+
+
+The source code of this application was fetched from [here](https://github.com/danieloh30/eap-session-replication). The only change we have made is in the [testHA.jsp](https://github.com/Azure/rhel-jboss-templates/blob/master/eap-session-replication/target/testHA.jsp) to add few images and provide few reference links. We have created a war file named [eap-session-replication.war](https://github.com/Azure/rhel-jboss-templates/blob/master/eap-session-replication/target/eap-session-replication.war) using these files and used this war file to deploy the sample application in the following Azure quickstart templates.
+
+* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-clustered-multivm-rhel" target="_blank"> JBoss EAP on RHEL (clustered VMs)</a>
+* <a href="https://github.com/Azure/azure-quickstart-templates/tree/master/jboss-eap-clustered-vmss-rhel" target="_blank"> JBoss EAP on RHEL (clustered VMSS)</a>
+
+### You can follow the documentation below to use this application in your environment.
+
+You will deploy a clustered JBoss Enterprise Application Server with a simple web application that demonstrates session replication across the cluster.
+
+This is a very common use case for enterprise web applications that require user sessions to remain alive even if the server
+they are being hosted on is restarted or suffers a fault.
+
+Documentation for JBoss EAP Clustering can be found [here](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/getting_started_with_jboss_eap_for_openshift_container_platform/#reference_clustering).
+
+To enable session clustering in your web application, EAP has to be configured (done by default in OpenShift) as well as adding the
+following to your **src/main/webapp/WEB-INF/web.xml**:
+
+     <distributable/>
+
+There are several major functionality differences in the OpenShift JBoss EAP 7 image that are documented
+[here](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/getting_started_with_jboss_eap_for_openshift_container_platform/#comparison_eap_and_xpaas_eap_image).
+
+The main difference for clustering being that `domain` mode is not supported - OpenShift controls the creation and distribution of applications in the containers.
+
+
+**Step 1 - Create Project**
```

Comment:
Next steps are for EAP on Openshift but it misses any information how to deploy application to instances created from EAP Azure templates. There should be provided link to such an documentation. It might continue from step. 4 with information how to check the deployment.

Reply:
I have made the required changes in the file which provides details and links on how to deploy the application and how to validate the deployment.